dev_tools/bash_scripts_test.py

```diff
@@ -283,7 +300,8 @@ def test_pytest_and_incremental_coverage_branch_selection():
     assert result.out == (
         'INTERCEPTED python '
         'dev_tools/run_pytest_and_incremental_coverage.py HEAD\n')
-    assert result.err == "Comparing against revision 'HEAD'.\n"
+    assert result.err.split('\n', 1)[0].split() == (
```

Comment:
Can we `strip()` instead of splitting?

Reply:
I looked into that and I don't think so, I get multiline string like 

Comparing against revision 'HEAD'.\n
Usage ...

and I need the first line somehow, `strip()` takes care only about leading and trailing characters.

Reply:
What is the exact output you are seeing?

What version of git are you using?

Reply:
I see exactly:
> Comparing against revision 'HEAD'.
> usage: find [-H | -L | -P] [-EXdsx] [-f path] path ... [expression]
>        find [-H | -L | -P] [-EXdsx] -f path [path ...] [expression]
> 
and
> $ git --version
> git version 2.19.0.605.g01d371f741-goog

Reply:
You're right, that was actually an error in the bash script. I didn't check what is actually going on here...

Reply:
Fixed